components/dialog/dialog.js

```diff
@@ -103,6 +108,7 @@ class Dialog extends LocalizeCoreElement(AsyncContainerMixin(DialogMixin(LitElem
 		this.width = 600;
 		this._handleResize = this._handleResize.bind(this);
 		this._handleResize();
+		this.describeContent = false;
```

Comment:
Nit: same thing here... let's move this up below `async` so that the property assignments are sorta alpha.